test/postgresConnectionStrings.test.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as assert from 'assert';
+import { addDatabaseToConnectionString } from '../src/postgres/postgresConnectionStrings';
```

Comment:
I think the build is failing because you have to import this from `extension.bundle` for tests to work when webpacked (which is how the build machine runs tests)

Reply:
Also need to export `addDatabaseToConnectionString` here: 
https://github.com/microsoft/vscode-cosmosdb/blob/master/extension.bundle.ts